scripts/qemu.py

```diff
@@ -33,7 +33,7 @@ def __init__(self, binary, args=[], wrapper=[], name=None, test_dir="/var/tmp",
         self._qemu_log_path = os.path.join(test_dir, name + ".log")
         self._popen = None
         self._binary = binary
-        self._args = list(args) # Force copy args in case we modify them
+        self.args = list(args) # Force copy args in case we modify them
```

Comment:
I think this is OK.  I don't have a strong opinion at this point about how much wrappers we should add to things such as the command line or use a different approach (like the templates in avocado-virt).  I feel that starting simple is the way to go, so +1 here.